### PR TITLE
bravebookmarks plugin

### PR DESCRIPTION
**Pull request by pacharanero**
The bookmarks file structure for Brave appears to be identical to Chrome and Chromium, which is not a surprise given Brave is built using Chromium. (Also worth noting that Vivaldi browser is also similarly built and so an Albert plugin for Vivaldi may well be able to be done in exactly the same way as this has been done)

This PR contains a carbon-copy of the Chrome bookmarks plugin, in which I have then edited all instances of 'Chrome' or 'Chromium' to 'Brave', using the appropriate case where necessary, and also updating things like file names, the CMakeLists.txt, and paths to executables.

I am using Brave installed as per the instructions here https://brave-browser.readthedocs.io/en/latest/installing-brave.html#linux . (Brave can also be installed using a Snap, but if you do this the Bookmarks file will not be in `~/.config` and Albert will not be able to find it.)

I am building Albert as per the instructions in the Albert documentation.

On my machine (Linux Mint 19.1) this works perfectly without any errors in the debug output.

image

I guess this plugin solves my immediate problem of wanting to use Albert with Brave. In the longer term, it might be possible and preferable to refactor (and rename?) the Chrome bookmarks plugin itself in order to make it work with any Chromium-based browser (Brave, Vivaldi, Chrome, Chromium, Canary, etc) but there would need to be additional UI elements in Albert to allow the user to select which browser they want to use for Albert, since many people will have more than one installed and will have a preference as to which browser's bookmarks Albert should scan, and which it should launch.

Thanks for making Albert!

### CODE REVIEW

**Comment by pacharanero** [comment]
Hi @ManuelSchneid3r just bumping this PR? Is it acceptable as it is? Would it need modifications? I'm still using Brave as my daily driver.

File: bravebookmarks/metadata.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "id" :              "org.albert.extension.bravebookmarks",
+    "name" :            "Brave bookmarks",
+    "version" :         "1.0alpha",
+    "platform" :        "Linux",
+    "group" :           "Extensions",
+    "author" :          "Manuel Schneider",
```
Comment (idkCpp) [inline bravebookmarks/metadata.json]: Is that the author though?

Comment (pacharanero) [reply bravebookmarks/metadata.json]: I can't in all honesty claim authorship for this code, it's literally a Find and Replace job on the Chrome plugin! :smile: Leave it as Manuel.

In the absence of an Albert Contributor Covenant: I'm happy to contribute this to the Albert project and I don't claim any copyright over the work. I assign ownership to the Albert project or to @ManuelSchneid3r , whichever is better for you.

**Comment by pacharanero** [comment]
Hi can I please bump this PR? Hoping to get this feature into Albert core if possible?

**Comment by ManuelSchneid3r** [comment]
Is this necessary? regarding the other PRs the chromium booksmarks plugin seems to do the job if we add the correct hardcoded strings

**Comment by pacharanero** [comment]
@ManuelSchneid3r it's a fair point - if you are happy to add these correct hardcoded strings to the Chromium plugin then yes there's no real need for a separate plugin.

**Comment by ManuelSchneid3r** [comment]
Okay thank you @pacharanero. Still we have to find the correct config paths since we have three options meanwhile

**Pull request closed without merging**